### PR TITLE
FWF-3447: [Bugfix] keycloak login page mobile responsiveness fixed

### PR DESCRIPTION
**Pull request by Ajay-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3447
Issue Type: BUG/ FEATURE

# Changes
keycloak login page mobile responsiveness fixed

# Screenshots (if applicable)
image
image
image
image
image

# Notes
>

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

**Review by shuhaib-aot — APPROVED** [review]
@Ajay-aot  please check register page also. and check with large screen

File: forms-flow-idm/keycloak/themes/formsflow/login/resources/css/login.css [hunk 1]
```diff
@@ -383,24 +383,22 @@
...
           text-align: left;
       }
   
       #kc-header-wrapper {
           font-size: 16px;
           font-weight: bold;
-          padding: 20px 60px 0 0;
           color: #72767b;
           letter-spacing: 0;
+          padding: 0 0 0;
```
Comment (shuhaib-aot) [inline forms-flow-idm/keycloak/themes/formsflow/login/resources/css/login.css:401]: you can simple give padding:0 right ?

**Comment by Ajay-aot** [comment]
> @Ajay-aot please check register page also. and check with large screen

image
image
image
image
image
@shuhaib-aot please check,register page is also looks fine

File: forms-flow-idm/keycloak/themes/formsflow/login/resources/css/login.css (hunk 1 above)
Comment (Ajay-aot) [reply forms-flow-idm/keycloak/themes/formsflow/login/resources/css/login.css]: yes,changed

**Review by arun-s-aot — APPROVED** [review]

**Pull request merged**